python/jmxquery/__init__.py

```diff
@@ -50,125 +60,156 @@ def __init__(self,
         self.metric_name = metric_name
         self.metric_labels = metric_labels
 
-    def to_query_string(self) -> str:
+    def to_query_string(self):
         """
         Build a query string to pass via command line to JMXQuery Jar
-
         :return:    The query string to find the MBean in format:
-
                         {mBeanName}/{attribute}/{attributeKey}
-
                     Example: java.lang:type=Memory/HeapMemoryUsage/init
         """
-        query = ""
+        query = []
         if self.metric_name:
-            query += self.metric_name
-
-            if ((self.metric_labels != None) and (len(self.metric_labels) > 0)):
-                query += "<"
-                keyCount = 0
-                for key, value in self.metric_labels.items():
-                    query += key + "=" + value
-                    keyCount += 1
-                    if keyCount < len(self.metric_labels):
-                        query += ","
-                query += ">"
-            query += "=="
-
-        query += self.mBeanName
+            query.append(self.metric_name)
+
+            if self.metric_labels != None:
+                if len(self.metric_labels) > 0:
+                    query.append("<")
+                    keyCount = 0
+                    for key, value in self.metric_labels.items():
+                        query.append("%s = %s" % (
+                            key,
+                            value
+                        ))
+                        keyCount += 1
+                        if keyCount < len(self.metric_labels):
+                            query.append(",")
```

Comment:
I think this loop can be improved a bit. The most simple change for better idiomatic Python is
```
total_count = len(self.metric_labels)
for count, (key, value) in enumerate(self.metric_labels.items(), 1):
    query.append("{} = {}".format(key, value))
    if count < total_count:
        query.append(",")
```

Reply:
Implemented per your recommendation.